.gitlab/hpsf-gitlab-ci.yml

```diff
@@ -48,6 +48,43 @@ NVIDIA-GH200:
     reports:
       junit: build/result_gh200.xml
 
+NVIDIA-P100:
+  extends: .common-setup
+  tags: [nvidia-p100]
+  image: nvcr.io/nvidia/cuda:12.3.2-devel-ubuntu22.04
+  rules:
+    - if: $CI_PIPELINE_SOURCE == "schedule"
+      variables:
+        CDASH_MODEL: "Nightly"
+    - when: on_success
+      variables:
+        CDASH_MODEL: "Experimental"
```

Comment:
We also set `CDASH_MODEL` in `before_script`.
```suggestion
```
but we can also clean this up in a follow-up together with GH200.

Reply:
Done